lib/src/callable/async_built_in.dart

```diff
@@ -99,8 +99,9 @@ class AsyncBuiltInCallable implements AsyncCallable {
 /// available as function [name] in built-in module [module].
 void warnForGlobalBuiltIn(String module, String name) {
   warnForDeprecation(
-      'Global built-in functions will be deprecated in the future.\n'
-      'Remove the --future-deprecation=global-builtin flag to silence this '
-      'warning for now.',
+      'Global built-in functions are deprecated and '
+      'will be removed in Dart Sass 3.0.0.\n'
```

Comment:
Style nit: only split strings at newlines or just before you'd hit the 80 char limit.